api/test/wfl/integration/source_test.clj

```diff
@@ -189,15 +192,40 @@
                 (is (= expected-num-records (count records))))
               (testing "all snapshot jobs were updated and corresponding snapshot ids were inserted"
                 (is (every? record-updated? records))))))
+        (let [stopped-source (source/update-source!
+                              (jdbc/with-db-transaction [tx (postgres/wfl-db-config)]
+                                (source/stop-source! tx source)
+                                (reload-source tx source)))]
+          (is (== expected-num-records (stage/queue-length stopped-source))
+              "no more snapshots should be enqueued")
+          (is (not (stage/done? stopped-source))
```

Comment:
Previously this line passed, but because `(stage/done? source)` was `nil`.  This method checks for the presence of a `stopped` attribute in the source, and so we must pass it the reloaded source after stopping it rather than the source as it was initially created.